go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/disintegration/imaging v1.6.0
 	github.com/fvbommel/sortorder v1.0.2
 	github.com/go-chi/chi v4.0.2+incompatible
+	github.com/gofrs/uuid v4.4.0+incompatible
```

Comment:
This allows for passing in a search query that can be a name or a stash id, and parse the text to find out which one.